pom.xml

```diff
@@ -114,6 +114,12 @@
       <scope>test</scope>
       <version>1.9.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.sonar</groupId>
```

Comment:
No idea why this is needed, it was just a guess. Because when i build the lts with “-X”, then i see more in the log:
>[ERROR] Failed to execute goal org.codehaus.sonar:sonar-packaging-maven-plugin:1.12.1:check-dependencies (default-check-dependencies) on project sonar-resharper-plugin: org.codehaus.sonar:sonar-plugin-api should be declared in dependencies -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal **org.codehaus.sonar:sonar-packaging-maven-plugin:1.12.1:check-dependencies** (default-check-dependencies) on project sonar-resharper-plugin: org.codehaus.sonar:sonar-plugin-api should be declared in dependencies
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)

Before my change i've got the same error locally like in the appveyor build. And after that dependency, this error was gone, and i've see just failing tests.

Therefore i've trought, that's this could solve that problem. But unfortunately appveyor has the next error, which i can't reproduce locally.

> maybe additional code adjustments are needed

Probably yes, to get rid of the old dependency.